modules/c++/sys/source/OSUnix.cpp

```diff
@@ -56,7 +56,7 @@ std::string readLink(const std::string& pathname)
 
     if (readlink(pathname.c_str(), buffer, PATH_MAX) == -1)
     {
-        throw except::Exception(Ctxt(strerror(errno)));
+        return "";
     }
```

Comment:
Then do you want to change line 389 to...

` if (!executableName.empty() && isFile(executableName))`
?

Also, in what situation is there actually a symlink in `/proc/self/exe` that `readlink()` fails on?